web/themes/custom/theme_cat/components/02-molecules/card-test/card-test.scss

```diff
@@ -0,0 +1,69 @@
+@import '../../imports.scss';
+
+.card-test {
+  width: 40rem;
+  height: 100%;
+  position: relative;
+  display: inline-block;
+  margin-bottom: $padding-large;
+  background-color: clr(off-white);
+  border-radius: $border-radius*3;
+  &:hover {
+    cursor: pointer;
+  }
+
+  &:focus-within,
+  &:active {
+    box-shadow: 0 0 0 0.4rem clr(bright-blue);
+  }
+
+  img {
+    width: 100%;
+    border-top-left-radius: $border-radius*3;
+    border-top-right-radius: $border-radius*3;
+  }
+
+  &__content {
```

Comment:
Ideally, we should not nest these elements as they don't need the specificity. It is better to separate them in a different line as `.card-test__content {`